security_monkey/decorators.py

```diff
@@ -160,18 +160,19 @@ def decorated_function(*args, **kwargs):
 
 def get_regions(account, service_name):
     if not service_name:
-        return None, ['us-east-1']
+        return None, [AWS_DEFAULT_REGION]
 
     sts = boto3.client('sts')
     role_name = 'SecurityMonkey'
     if account.getCustom("role_name") and account.getCustom("role_name") != '':
         role_name = account.getCustom("role_name")
 
-    role = sts.assume_role(RoleArn='arn:aws:iam::' + account.identifier + ':role/' + role_name, RoleSessionName='secmonkey')
+	arn = ARN_PREFIX + ':iam::' + account.identifier + ':role/' + role_name
+    role = sts.assume_role(RoleArn=arn, RoleSessionName='secmonkey')
```

Comment:
`arn`  is undefined when the if-statement false through.

Line 170 is likely tabbed over once to many times.  It should be at the same indentation level as line 171.